node/reqres.js

```diff
@@ -47,13 +48,29 @@ function TChannelIncomingRequest(id, options) {
     self.headers = options.headers || {};
     self.checksum = options.checksum || null;
     self.checksumType = options.checksumType || 0;
-    self.arg1 = options.arg1 || emptyBuffer;
```

Comment:
Because they're going away real soon at this layer, to be provided by the upcoming use of argstream as base class in a future pr.